spec/classes/nginx_spec.rb

```diff
@@ -360,9 +360,9 @@
             it do
               is_expected.to contain_file('/var/log/nginx').with(
                 ensure: 'directory',
-                owner: 'www-data',
+                owner: 'root',
                 group: 'adm',
-                mode: '0750'
+                mode: '0755'
```

Comment:
This test is failing on RHEL-based tests.

Reply:
Looks like that's a valid failure because I copy-pasted an incorrect default